### PR TITLE
chore: Fields added to erpnext file "purchase_incoice_item.json"

### PR DESCRIPTION
**Pull request by mrunmay-epi**
Task link: https://bloomstack.com/desk#Form/Task/TASK-2021-00250
Bloomstack core PR: https://github.com/Bloomstack/bloomstack_core/pull/638

Added fields to erpnext "purchase_incoice_item.json" file.

Before:
image

After:
image

### CODE REVIEW

File: erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json [hunk 1]
```diff
@@ -67,6 +67,12 @@
   "total_weight",
   "column_break_38",
   "weight_uom",
+  "cultivation_tax_details_section",
```
Comment (sahil28297) [inline erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json:70]:
```suggestion
  "cultivation_tax_details",
```

File: erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json [hunk 2]
```diff
@@ -827,11 +833,41 @@
    "fieldtype": "Long Text",
    "label": "Label Print Details",
    "read_only": 1
+  },
+  {
+   "fieldname": "cultivation_tax_details_section",
```
Comment (sahil28297) [inline erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json:838]:
```suggestion
   "fieldname": "cultivation_tax_details",
```

**Review by sahil28297 — CHANGES_REQUESTED** [review]
The Property setters are missing!!

File: erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json (hunk 1 above)
Comment (mrunmay-epi) [reply erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json:70]: Hi Sahil,

Updated the PR as per suggestions.

PR link: https://github.com/Bloomstack/erpnext/pull/1603

**Comment by sahil28297** [comment]
Duplicate

**Pull request closed without merging**